101-synapse-poc/nestedtemplates/logicapproleassignments.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "logicAppPauseName": {
+            "type": "string"
+        },
+        "logicAppResumeName": {
+            "type": "string"
+        }
+    },
+    "variables": {
+        "contributor": "b24988ac-6180-42a0-ab88-20f7382dd24c"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Authorization/roleAssignments",
+            "apiVersion": "2020-04-01-preview",
+            "name": "[guid(parameters('logicAppPauseName'))]",
+            "properties": {
```

Comment:
add the principalType property for both roleAssignments

Reply:
added principalType specifying msi

Reply:
added principalType specifying ServicePrincipal. msi was not the correct value.